src/plugins/python/tiledbinding.py

```diff
@@ -77,7 +77,11 @@
 cls_tileset.add_method('tileHeight', 'int', [])
 cls_tileset.add_method('tileSpacing', 'int', [])
 cls_tileset.add_method('margin', 'int', [])
-#cls_tileset.add_method('tileOffset', 'QPoint', [])
+cls_tileset.add_method('tileOffset', 'QPoint', [])
+cls_tileset.add_method('gridSize', 'Qsize', [])
+cls_tileset.add_method('setGridSize', None, ('Qsize', 'gridSize'))
```

Comment:
Should be `QSize` (though currently this class is not defined, nor is `QPoint`, which is probably why the `tileOffset` method is commented out).